hil/cli.py

```diff
@@ -288,6 +288,14 @@ def serve_networks():
         sleep(sleep_time)
 
 
+@cmd
+def list_create():
+    """List all users"""
+    q = C.users.list()
```

Comment:
this would be C.`user` not `users`

Reply:
this reminds me of creating a unit test for this api ;)Thanks